Documentation/BuildInstructions.md

```diff
@@ -112,6 +112,19 @@ To add a package from the ports collection to Serenity, for example curl, change
 run `./package.sh`. The source code for the package will be downloaded and the package will be built. The next time you
 start Serenity, `curl` will be available.
 
+Ports might also have additional dependencies. Most prominently, you may need:
+autoconf, automake, bison, flex, gettext, gperf, help2man, imagemagick (specifically "convert"),
+libgpg-error-dev, libtool, lzip, meson, nasm (or another assembler), qt6-base-dev, rename, zip.
+
+For select ports you might slightly more exotic dependencies such as:
```

Comment:
This sentence appears to be missing words? `select ports you might slightly more`?

Reply:
Oh! Yes, absolutely, I accidentally a word.